pandas/compat/numpy/__init__.py

```diff
@@ -11,6 +11,8 @@
 _np_version_under1p16 = _nlv < LooseVersion("1.16")
 _np_version_under1p17 = _nlv < LooseVersion("1.17")
 _np_version_under1p18 = _nlv < LooseVersion("1.18")
+_np_version_under1p19 = _nlv < LooseVersion("1.19")
+_np_version_under1p20 = _nlv < LooseVersion("1.20")
```

Comment:
I think this is targeted for NumPy 1.20, but verifying in https://github.com/numpy/numpy/pull/16554#issuecomment-643232285.